src/App.tsx

```diff
@@ -1,26 +1,38 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useTodos } from './app/hooks';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const { currentTodo, handleSetTodos } = useTodos();
+  const [isLoading, setIsLoading] = useState(true);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    getTodos()
+      .then(handleSetTodos)
+      .finally(() => setIsLoading(false));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
The `useEffect` hook has an empty dependency array, which means it will only run once when the component mounts. If `handleSetTodos` is a function that could change over time or relies on other state or props, it should be included in the dependency array to avoid potential issues. Ensure that this is the intended behavior.